isomorphe/batch.py

```diff
@@ -67,6 +67,7 @@ class SuccessTransformBatchRecord(TransformBatchRecord):
     result: bytes
     info: str
     log: TransformLog | None = None
+    has_diff: bool = True
```

Comment:
I'm not super fond of adding this, but I'm not sure we want to add something like a FakeSuccess class for that?

Reply:
LGTM